Sources/Support/Windows/Stringify.cpp

```diff
@@ -56,6 +56,8 @@ char const *Stringify::ExceptionCode(DWORD code) {
     DO_STRINGIFY(EXCEPTION_STACK_OVERFLOW)
   case DS2_EXCEPTION_UNCAUGHT_USER:
     return "0xE06D7363 (uncaught user exception)";
+  case DS2_EXCEPTION_UNCAUGHT_COM:
```

Comment:
You should put this before `DS2_EXCEPTION_UNCAUGHT_USER` (to follow the order from `ExtraWrappers.h`).